### ISSUE TITLE
Couple of small changes please:

### ISSUE THREAD

**Issue by JunglePredator** [hql287/Manta#394]
Couldn't find any issues opened about these so hopefully I am not creating a duplicate.

First off it seems mandatory to have an email for the target of the invoice? Least I can't see a way to turn it off & the program gives an error if it's left blank. It'd be nice to be able to leave it blank/turn off as I don't deal with online contact of customers & have no need to ask them this... and it makes it look unprofessional if I insert some dummy email.
To further that... it'd be nice if the program accepted *any* field left blank & just didn't display them on the final copy & instead of giving an unpassable error only gave a suggestion like 'are you sure you want to leave this blank? it won't show up'.

2nd the program doesn't allow negative amounts to stand for item values; it'd be nice if it did to show a transfer/discount of $ in the opposite way. Yes I know there's a bulk discount but sometimes a negative line value for one specific item is needed.

Lastly it'd be nice to be able to disable it's attempts to call home. I've firewalled the program but it still throws 2 error boxes when it realises it cannot contact the internet.

Thanks!

**Comment by Dani2449** [comment]
yes, i thought the same. take out the mandatory field, especially email. and to be able to add the united type there are different if it will be able to set this in settings.
also to be able to add products.
also to be able to add some filed customized.
